CONTRIBUTING.md

```diff
@@ -47,7 +47,7 @@ requests_).
  - Tim Whitcomb
  - (Scott Wales)
  - Tomek Trzeciak
- - (Thomas Coleman)
+ - Thomas Coleman
```

Comment:
Good spotting :+1:

Reply:
I almost didn't look, but then I figured I really should follow the checklist and not just tick things off.